benchmark/Cpp/Savina/src/concurrency/SleepingBarber.lf

```diff
@@ -0,0 +1,279 @@
+/**
+ * Copyright (C) 2020 TU Dresden
+ * 
+ * In this benchmark, there are three main components: The barber, a waiting room and the customers.
+ * The customers are produced and controlled by a CustomerFactory.
+ * This implementation is close to the Savina Akka implementation
+ * in terms of the work that is done in each actor/reactor and
+ * that for each message sent in Savina there is a message send
+ * in this implementation. But this implementation does not
+ * show the basic idea of the problem in that the waiting
+ * room is basically never because the customers arrival is
```

Comment:
Wonky sentence.

Reply:
In fact the whole comment is outdated. I think this is (in part) the old comment that Hannes wrote and I forgot to update it. It's fixed now.